keyboards/eco/readme.md

```diff
@@ -0,0 +1,9 @@
+ECO
+===
+
+![ECO](http://i.imgur.com/YligKxr.jpg)
+
+An economical 4x14 ortholinear keyboard
+
+Design and concept by : BishopKeyboards
+PCB by : u/That-Canadian
```

Comment:
Would you mind updating your readme so that it follows [this format](https://github.com/qmk/qmk_firmware/issues/1362#issuecomment-310859229)?